docs/releases/2.0.txt

```diff
@@ -611,6 +611,11 @@ Miscellaneous
 
 * Support for GEOS 3.3.x is dropped.
 
+* The way data is selected for ``GeometryField`` is changed to improve
+  performance. If you don't already wrap geometry fields with
+  ``connection.ops.select`` in raw SQL queries, you'll need to add that. See
```

Comment:
@timgraham, I'm not sure if there is sense to mention ``connection.ops.select`` here since it wasn’t doc'd anyhow so I don't think someone used it. Perhaps we should say without any condition: "your raw SQL queries need to be updated".

Reply:
Good point, I've amended it.